_data/students.yml

```diff
@@ -619,6 +619,64 @@
 
 - name: mercuryprasad
   github: mercuryprasad
-  image: mercuryprasad.jpg
+  image: mercuryprasad.jpeg
   twitter: prasad_pitagi
   bio: Loves electronics, learning coding
+
+- name: Kriya 
+  facebook: Akshata Kriya Betageri
+  image: kriya.jpg
+  twitter: Debena2
+  bio: It is always the adventurous who accomplish great things!
+
+- name: Anushri
+  image: anushri.jpg
+  twitter: Uanushri
+  bio: In search of purpose of existance
+
+- name: Akshata
+  image: akshata.jpg
+  twitter:swararevankar 
```

Comment:
Include a space after `twitter:` and before  `swararevankar`. Kindly follow the pattern :)